k8s/demo/common/ccd/latest-ccd-chart.yaml

```diff
@@ -74,14 +74,25 @@ spec:
           ADMINWEB_WHOAMI_URL: '{{ .Values.global.ccdDefinitionStoreUrl }}/api/idam/profile'
           ADMINWEB_IMPORT_AUDITS_URL: '{{ .Values.global.ccdDefinitionStoreUrl }}/api/import-audits'
           ADMINWEB_AUTHORIZATION_URL: '{{ .Values.global.ccdDefinitionStoreUrl }}/api/idam/adminweb/authorization'
-    ccd-case-activity-api:
-#      TODO this has changed to activityApi
-      enabled: true
-    ccd-case-print-service:
-      enabled: true
-      nodejs:
-        ingressClass: 
-        ingressHost: 
+    
+    # CCD Supporting services
+    ccd:
+      printService:
+        enabled: true
+        nodejs:
```

Comment:
1) are you sure having `nodejs:` nested here will be picked up? I don't think
2) much of this info is already present in the chart-ccd. Ideally we should add the config there rather than here. If you want to add that here to test it more quickly, please add a TODO comment where you mention to move all this to the chart-ccd once is proved working.
Remember we want as little config as possible in flux

Reply:
it is right config Mario, as those url change in demo, hence override.